indico/htdocs/js/indico/jquery/clearableinput.js

```diff
@@ -69,7 +69,7 @@
             var self = this;
             self.element.val(self.options.emptyvalue).trigger('propertychange').trigger('change');
             self._refreshClearIcon();
-            self.options.onClear();
+            self.options.onClear.call(self.element);
```

Comment:
that is right, `self.element` seems to be much more useful to the handlers (e.g. I can retrieve the value inputted in the field)